discovery-provider/src/tasks/index_nethermind.py

```diff
@@ -829,6 +829,12 @@ def revert_blocks(self, db, revert_blocks_list):
                 .all()
             )
 
+            revert_playlist_seen = (
+                session.query(PlaylistSeen)
+                .filter(PlaylistSeen.blocknumber == revert_block_number)
+                .all()
```

Comment:
for my own knowledge, the diff between block number and the revert is never going to be too high right?

Reply:
yea we only revert up to 50 blocks or so and but pretty sure with ACDC it's just 1 block at a time